vue-components/src/components/LookupInput.vue

```diff
@@ -109,6 +118,10 @@ export default defineComponent( {
 			type: String,
 			default: '',
 		},
+		label: {
```

Comment:
Yes! Thanks for catching this.

Reply:
The QuantityInput already has an aria-label.

Reply:
> The `QuantityInput` already has an aria-label.

Yes, this makes this somewhat confusing. But that aria-label is for the actual `<input>` because that component does not have a proper label (this is what the `Lookup` component adds).

On the other hand, the value coming from the  `:label="..."` prop will be used on the QuantityInput.

Reply:
I added the label because otherwise there is an error when running the tests.

Reply:
This is the error. 

![image](https://user-images.githubusercontent.com/3005239/171035757-2ea6a8af-3de5-48a4-9e90-aba7df497c45.png)